doc/api/worker_threads.md

```diff
@@ -61,6 +61,26 @@ Worker threads inherit non-process-specific options by default. Refer to
 [`Worker constructor options`][] to know how to customize worker thread options,
 specifically `argv` and `execArgv` options.
 
+## `worker.connect(target[, data][, timeout])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `target` {number} The target thread id.
+* `data` {any} Any arbitrary, cloneable JavaScript value.
+* `timeout` {number} Time to wait in milliseconds. By default it's `undefined`, which
+  means wait forever.
```

Comment:
This should clarify what the timeout is waiting for.

Reply:
Added!